### ISSUE TITLE
License text is out of place in SVG sprites

### ISSUE THREAD

**Issue by javier-godoy** [FortAwesome/Font-Awesome#17214]
Up to FontAwesome 5.14.0 the SVG sprites contained the license in the prolog, immediately after the XML declaration:

```
<?xml version="1.0" encoding="UTF-8"?>
<!--
Font Awesome Free 5.14.0 by @fontawesome - https://fontawesome.com
License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License)
-->
```

In version 5.15.0, the license was moved under the `<symbol>` element
```
<?xml version="1.0" encoding="UTF-8"?>
<svg xmlns="http://www.w3.org/2000/svg" style="display: none;">
  <symbol id="ad" viewBox="0 0 512 512">
    
<!--
Font Awesome Free 5.15.0 by @fontawesome - https://fontawesome.com
License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License)
-->
```

**To Reproduce**
view-source:https://raw.githubusercontent.com/FortAwesome/Font-Awesome/5.14.0/sprites/solid.svg
view-source:https://raw.githubusercontent.com/FortAwesome/Font-Awesome/5.15.0/sprites/solid.svg

Whatever you do, don't click the red button.

**Expected behavior**
The license text should be located before the actual contents, so that it's clear that it covers the whole file. (Actually, I'm postprocessing the file and having the license in a consistent position -as it was until 5.14.0- would facilitate preserving the copyright notice)

**Version and implementation**
Version: 5.15.0
Browser and version: does not apply

- [ ] SVG with JS
- [ ] Web Fonts with CSS
- [x] SVG Sprites
- [ ] On the Desktop

**Comment by tagliala** [comment]
Hi!

Thanks for being part of the Font Awesome Community and thanks for reporting this

Let's assign @robmadole

**Comment by robmadole** [comment]
Thanks for the report. We'll get this fixed up!

**Comment by robmadole** [comment]
We've got this fixed in our build system and it will be released with 5.15.11 on Monday.

**Comment by javier-godoy** [comment]
Thanks. The position of the license text in 5.15.1 sprites is the same as in 5.14.0
